### ISSUE TITLE
ResourceOf need an extra ctor with Class argument

### ISSUE THREAD

**Issue by yegor256** [yegor256/cactoos#1517]
Now I have to do this:

```java
class Foo {
  new ResourceOf("foo.txt", Foo.class.getClassLoader())
}
```

I believe it would be more convenient if we would have an extra ctor:

```java
class Foo {
  new ResourceOf("foo.txt", Foo.class)
}
```

**Comment by yegor256** [comment]
@victornoel this PR should fix it: #1520

**Comment by yegor256** [comment]
@victornoel are you planning to release 0.49 any time soon?

**Comment by victornoel** [comment]
@yegor256 I would first like to have #1528 merged at least, but there are no REV available, would you interested in reviewing the PR? Sorry for the late reply :)

**Comment by victornoel** [comment]
@yegor256 closing this, 0.49 will be released in #1530 in a few minutes

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#1517` is not assigned, can't get performer
